### ISSUE TITLE
external library proguard configuration should not influence to client application.

### ISSUE THREAD

**Issue by hatched-mariusz** [RadiusNetworks/flybuy-android#20]
your core-relase.aar has a proguard rules in the proguard.txt file.
These rules are automatically merged with client application rules when the library is added to the client. 
One of your rule is: 

```
# Keep package names to prevent a.a name collisions
-keeppackagenames
```
Which is merged with the client proguard rules and now this rule is the official client rule and the app keeps packages names that are wrong
the correct should be: 
```
# Keep package names to prevent a.a name collisions
-keeppackagenames com.radiusnetworks.flybuy.sdk
```

**Comment by botnerd** [comment]
Thanks for reporting the issue. We'll look into and get it updated.

**Comment by botnerd** [comment]
This has been fixed in the `2.8.0` release of the SDK. If you have any additional issues, please let us know.
